apps/utr-frontend/components/package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "components",
+	"version": "1.0.0",
+	"description": "React.js Components library",
+	"author": "luis-rieke",
+	"homepage": "",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"source": "lib/index.js",
+	"directories": {
+		"lib": "lib",
+		"test": "__tests__"
+	},
+	"files": [
+		"lib"
+	],
+	"scripts": {
+		"test": "echo \"Error: run tests from root\" && exit 1",
+		"dev": "microbundle watch --jsx React.createElement lib/*.js",
+		"storybook": "start-storybook -p 6006",
+		"build-storybook": "build-storybook"
+	},
+	"devDependencies": {
+		"@babel/core": "^7.15.8",
+		"@storybook/addon-actions": "^6.3.12",
+		"@storybook/addon-essentials": "^6.3.12",
+		"@storybook/addon-links": "^6.3.12",
+		"@storybook/react": "^6.3.12",
+		"babel-loader": "^8.2.3",
+		"microbundle": "^0.14.1"
+	},
+	"dependencies": {
+		"@types/react": "^17.0.33",
+		"@types/react-dom": "^17.0.10",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2"
+	}
+}
```

Comment:
Kannst du die noch rausnehmen, hier brauchen wir keine Package json

Reply:
habe ein paar Rückfragen, sind in Slack